recipes/client.rb

```diff
@@ -18,6 +18,14 @@
 # limitations under the License.
 #
 
+nt_version = ::Windows::VersionHelper.nt_version(node)
+
+if nt_version == 6.1
```

Comment:
Absolutely, I will change it to that and test again.
http://giphy.com/gifs/star-wars-181OUQHOCfde0